ny-portal/src/app/(frontend)/komponenter/page.tsx

```diff
@@ -8,12 +9,22 @@ export default async function Components() {
 
     const data = await payload.find({
         collection: "component-page",
+        select: {
+            title: true,
+            slug: true,
+        },
     });
 
     return (
-        <ul>
+        <ul style={{ display: "flex", flexDirection: "column", gap: "1rem" }}>
```

Comment:
Her burde det vel gå an å bruke `Flex`-komponenten vår? Hva med noe som dette:

```suggestion
        <Flex as="ul" direction="column" gap={16}>
```